package.json

```diff
@@ -101,7 +101,6 @@
     "@types/node": "14.11.7",
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
-    "flexboxgrid": "6.3.1",
```

Comment:
should removing make by another PR?

Reply:
flexboxgrid was only used by platform forms, and this PR gives a replacement for it. Does it really need another PR just to remove this lib usage?